vishwamai/models/gpu/transformer.py

```
@@ -14,6 +14,7 @@
     DeepGEMMGroupedLinear,
     get_optimal_kernel_config
 )
+from vishwamai.models.gpu.optimizations.deep_ep.utils import init_expert_parallel
```

Ruff (0.8.2)

`vishwamai.models.gpu.optimizations.deep_ep.utils.init_expert_parallel` imported but unused

Remove unused import: `vishwamai.models.gpu.optimizations.deep_ep.utils.init_expert_parallel`

(F401)